tests/tests/datm_cdeps_mx025_cfsr

```diff
@@ -10,15 +10,18 @@ export LIST_FILES="RESTART/MOM.res.nc \
                    RESTART/MOM.res_1.nc \
                    RESTART/MOM.res_2.nc \
                    RESTART/MOM.res_3.nc \
-                   RESTART/iced.2011-10-02-00000.nc \
-                   RESTART/DATM_CFSR.cpl.r.2011-10-02-00000.nc"
+                   RESTART/iced.2011-10-01-43200.nc \
+                   RESTART/DATM_CFSR.cpl.r.2011-10-01-43200.nc"
 
 export_datm_cdeps
-export WLCLK=40
 export DATM_SRC="CFSR"
 export FILENAME_BASE='cfsr.'
 export IATM=1760
 export JATM=880
+export DT_DYNAM_MOM6='900'
```

Comment:
I don't quite understand why the two tests aren't using the same configuration as the cpld_control_c384. In that case, the default timesteps are set for the 1deg case in default_vars. But for the 1/4deg case, we re-set the timesteps like so:

export DT_DYNAM_MOM6='900'
export DT_THERM_MOM6='1800'
export CPL_SLOW=${DT_THERM_MOM6}
export CPL_FAST=${DT_ATMOS} 

It seems you'd still be using the 1deg settings (3600s) for the CPL_SLOW and the DT_THERM_MOM6.

Reply:
@DeniseWorthen @junwang-noaa:  I will update the following 4 tests so that the same DT_THERM_MOM6 (=900) and DT_DYNAM_MOM6(=1800) are used:

1. datm_mx025_cfsr
2. datm_mx025_gefs
3. datm_cdeps_mx025_cfsr
4. datm_cdeps_mx025_gefs